### PR TITLE
[IMP]product_packaging_multi_barcode: Display barcode packaging

### PR DESCRIPTION
**Pull request by cyrilmanuel**
Display barcode of packaging product or product if not present.

### CODE REVIEW

File: product_packaging_multi_barcode/models/product_barcode.py [hunk 1]
```diff
@@ -14,6 +14,31 @@ class ProductBarcode(models.Model):
         ondelete="cascade",
     )
 
+    display_product_id = fields.Many2one(
```
Comment (rousseldenis) [inline product_packaging_multi_barcode/models/product_barcode.py:17]: @cyrilmanuel Thanks for this. I would have added it to another module instead.

Comment (cyrilmanuel) [reply product_packaging_multi_barcode/models/product_barcode.py:17]: @rousseldenis  thanks, do you think it shall be added into a new module ? or you have an idea on which one ?

Comment (cyrilmanuel) [reply product_packaging_multi_barcode/models/product_barcode.py:17]: @rousseldenis gentle ping :  any idea or can we merge it like this ?

Comment (rousseldenis) [reply product_packaging_multi_barcode/models/product_barcode.py:17]: @cyrilmanuel Ok, didn't see the packaging_id definition. We can move forward.

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@cyrilmanuel Could you attract a reviewer here ?

**Review by wouitmil — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]
Maybe reformat the commit line so that it fits (https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#commit-message)

**Comment by sebalix** [comment]
@cyrilmanuel better to not include the Odoo version number in the commit message, as commits can be ported to other versions this is misleading and doesn't help commit comparison afterwards

**Pull request merged**